recipes/pycovjson/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pycovjson" %}
+{% set version = "0.3.9" %}
+{% set sha256 = "f1482516307189f8b7b8b0f3eeea645923298cea2bfb67f43f0d5e29fa2cc504" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
It looks like this package has entry points. Please include them:

https://github.com/Reading-eScience-Centre/pycovjson/blob/master/setup.py#L33
https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

Reply:
ack, done